server/src/models/people/schema.js

```diff
@@ -1,534 +1,79 @@
-
-const common_org_and_gov_queries = `
-  employee_type_data(dimension: Type = all_dimensions): HeadcountData
-  employee_type_info: EmployeeTypeInfo
-  employee_region_data(dimension: Region = all_dimensions): HeadcountData
-  employee_age_group_data(dimension: AgeGroup = all_dimensions): HeadcountData
-  employee_ex_lvl_data(dimension: ExLvl = all_dimensions): HeadcountData
-  employee_gender_data(dimension: Gender = all_dimensions): HeadcountData
-  employee_fol_data(dimension: Fol = all_dimensions): HeadcountData
-`;
-
 const schema = `
-  extend type Org {
-    ${common_org_and_gov_queries}
-  }
-  extend type Gov {
-    ${common_org_and_gov_queries}
-  }
-
-  enum Type {
-    all_dimensions
-    cas
-    ind
-    na
-    stu
-    ter
-  }
-  enum Region {
-    all_dimensions
-    ab
-    bc
-    mb
-    nt
-    ncr
-    nb
-    nl
-    ns
-    nu
-    onlessncr
-    abroad
-    pe
-    qclessncr
-    sk
-    na
-    yt
+  extend type Org{
+    employee_summary: OrgEmployeeSummary
   }
-  enum AgeGroup {
-    all_dimensions
-    age20less
-    age20to24
-    age25to29
-    age30to34
-    age35to39
-    age40to44
-    age45to49
-    age50to54
-    age55to59
-    age60to64
-    age65plus
-    na
-  }
-  enum ExLvl {
-    all_dimensions
-    ex1
-    ex2
-    ex3
-    ex4
-    ex5
-    non
-  }
-  enum Gender {
-    all_dimensions
-    female
-    male
-    sup
-    na
-  }
-  enum Fol {
-    all_dimensions
-    eng
-    fre
-    sup
-    na
-  }
-
-  type HeadcountData {
-    data: [HeadcountDatum]
-    record: [HeadcountRecord]
-    avg_share: [HeadcountShare]
-    basic_trend: BasicHeadcountTrend
-  }
-  type BasicHeadcountTrend {
-    first_active_year: String
-    last_active_year: String
-    active_year_count: Float
 
-    avg_share: Float
-    avg_headcount: Float
-    change_percent: Float
-    ppl_last_year_5: Float
-    ppl_last_year_4: Float
-    ppl_last_year_3: Float
-    ppl_last_year_2: Float
-    ppl_last_year: Float
-  }
-  type HeadcountRecord {
-    dimension: String
-    ppl_last_year_5: Float
-    ppl_last_year_4: Float
-    ppl_last_year_3: Float
-    ppl_last_year_2: Float
-    ppl_last_year: Float
-  }
-  type HeadcountShare {
+  extend type Gov {
+    employee_summary: GovEmployeeSummary
+  }
+  
+  type OrgEmployeeSummary {
+    org_id: String
+    employee_age_group: [EmployeePopData]
+    employee_ex_lvl: [EmployeePopData]
+    employee_gender: [EmployeePopData]
+    employee_fol: [EmployeePopData]
+    employee_region: [EmployeePopData]
+    employee_type: [EmployeePopData]
+    employee_avg_age: [EmployeeAvgAge]
+  }
+
+  type GovEmployeeSummary {
+    id: String
+    employee_age_totals: [EmployeeGovData]
+    employee_ex_lvl_totals: [EmployeeGovData]
+    employee_gender_totals: [EmployeeGovData]
+    employee_fol_totals: [EmployeeGovData]
+    employee_region_totals: [EmployeeGovData]
+    employee_type_totals: [EmployeeGovData]
+    employee_gov_avgs: [EmployeeGovAvgs]
+  }
+  type DataObjectWithDim {
```

Comment:
Let's be a bit more descriptive like `EmployeeDataWithDimension` and `EmployeeDataWithoutDimension`